.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,29 @@
+<!-- Please feel free to remove whatever sections/lines in this aren't relevant. -->
+
+## Title Line Template: [Website] - [UI components]: [Brief description.]
```

Comment:
@maya If it's much likelier that people will only be addressing one component in a given issue, then I think using [UI component] makes more sense.